src/runtime/offers-flow.js

```diff
@@ -121,17 +121,19 @@ export class OffersFlow {
     this.skus_ = feArgsObj['skus'] || [ALL_SKUS];
 
     /** @private @const {!Promise<!ActivityIframeView>} */
```

Comment:
the type needs to be updated to reflect that the Promise can return null (!Promise<?ActivityIframeView>)